sql/core/src/test/scala/org/apache/spark/sql/execution/QueryExecutionSuite.scala

```diff
@@ -232,7 +234,7 @@ class QueryExecutionSuite extends SharedSparkSession {
 
   test("Logging plan changes for execution") {
     val testAppender = new LogAppender("plan changes")
-    withLogAppender(testAppender) {
+    withLogAppender(testAppender, level = Some(Level.INFO)) {
```

Comment:
In the cases of `QueryExecutionSuite`, `AdaptiveQueryExecSuite`, and `PythonDataSourceSuite`, similar modifications will be submitted as a separate PR. Currently, there is a tight coupling between the design of test cases and the configuration of `rootLogger.level` in the `log4j2.properties` file.